Resources/Prototypes/Accents/full_replacements.yml

```diff
@@ -59,6 +62,9 @@
   - accent-words-zombie-5
   - accent-words-zombie-6
   - accent-words-zombie-7
+  - accent-words-zombie-8
+  - accent-words-zombie-9
+  - accent-wors-zombie-10
```

Comment:
Missing `accent-words-cat-6` and `accent-words-cat-7` in this file

Reply:
damn, i forgot about adding more cat stuff

Reply:
all done